testing/test_mark.py

```diff
@@ -37,16 +40,16 @@ class SomeClass:
         assert pytest.mark.foo.with_args(SomeClass) is not SomeClass  # type: ignore[comparison-overlap]
 
     def test_pytest_mark_name_starts_with_underscore(self):
-        mark = Mark()
+        mark = MarkGenerator()
         with pytest.raises(AttributeError):
             mark._some_name
 
 
-def test_marked_class_run_twice(testdir):
+def test_marked_class_run_twice(pytester: Pytester):
```

Comment:
To complete the type annotation, please also add `-> None`. Otherwise mypy still considers it incomplete.